### DISCUSSION TITLE
FOS-PHP Web UI for Boxedwine

### DISCUSSION THREAD

**Discussion by hitsfm** [danoon2/Boxedwine#61] [Ideas]
Good day all. I worked on FOS-PHP script. It is a nice polished web UI that looks and runs very much like the Windows 3.1 program manager. It uses boxedwine to launch windows apps and wraps then in nice draggable UI windows. Let me know what you think. I'm new to all this! 
https://github.com/hitsfm/FOS-PHP
[](https://github.com/hitsfm/FOS-PHP)

**Comment by danoon2** [comment]
Thanks for sharing.  I always like to see the different ways that people use Boxedwine.
